constants/constants.go

```diff
@@ -57,3 +58,11 @@ var ErrInternal = errors.New("internal error")
 
 // TestMaxContentLength is the max Ethereum RPC Content-Length used in tests
 var TestMaxContentLength = 1024 * 512
+
+// UnlimitedExpirationTime is the maximum value for uint256, which mens there is
+// effectively no limit on the maximum expiration time for orders.
+var UnlimitedExpirationTime *big.Int
+
+func init() {
+	UnlimitedExpirationTime, _ = big.NewInt(2).SetString("115792089237316195423570985008687907853269984665640564039457584007913129639935", 10)
```

Comment:
Can you add a comment to what this number is? e.g., `2^64-1`?

Reply:
I already included a comment at the declaration of `UnlimitedExpirationTime` just above (the correct place for godoc comments). But I'll add that this is equal to 2^256-1.